src/io/probst/idea/clangformat/ClangFormatAction.java

```diff
@@ -165,10 +158,7 @@ public String call() throws Exception {
   private ProcessBuilder getCommand(String filePath, int cursor, int selectionStart,
       int selectionLength) throws ExecutionException, InterruptedException {
     Settings settings = Settings.get();
-
-    ProcessBuilder command = new ProcessBuilder().command(settings.clangFormatBinary, "-style=file",
-        "-output-replacements-xml", "-assume-filename=" + filePath, "-cursor=" + cursor,
-        "-offset=" + selectionStart, "-length=" + selectionLength);
+    ProcessBuilder command = new ProcessBuilder().command(settings.clangFormatBinary,filePath, "-style=file", "-cursor=" + cursor);
```

Comment:
All right. I didn't find a quick solution. That's why I wrote it as a temporary solution.